Makefile

```diff
@@ -1042,6 +1045,21 @@ geojson/albers/city-labels.geojson:
 				name:name \
 		> $@
 
+tiles/wapo-2016-election-centroids.mbtiles: geojson/albers/centroid-states.geojson \
+	geojson/albers/centroid-counties.geojson \
+	geojson/albers/centroid-districts.geojson
+	mkdir -p $(dir $@)
+	tippecanoe --projection EPSG:3857 \
+		-f \
+		--named-layer=states:geojson/albers/centroid-states.geojson \
```

Comment:
let's name these `state-centroids`, `county-centroids`, `district-centroids` (or something like it) -- the reason being, if we make sure that there are no layer-name collisions, we can use Mapbox's vector-tile "compositing", where they combine VTs from different tilesets on the server side if you use a url like `mapbox://devseed.tileset-one,devseed.tileset-two`.